_docs/reference/release-roadmap.md

```diff
@@ -15,6 +15,7 @@ Istio 0.2 will be our second release, and will work across multiple Kubernetes c
 - Support for VMs joining an Istio Mesh.
 - Fully-automated injection of Envoy proxies into Kubernetes pods.
 - Standard extension points for addons and plugins.
+- Expanded service-to-service authentication support for VMs.
```

Comment:
"Expanded authorization with callouts to external services." is for integration with external authorization system. This item is about adding authentication support for services running on VMs (non-k8s environment). As Sven said, authn is already included as part of "Support for VMs joining an Istio Mesh" item above. We may leave this as is, or update this item to include clarify which features are included